runtime/doc/editing.txt

```diff
@@ -1236,29 +1244,50 @@ present in 'cpoptions' and "!" is not used in the command.
 							*:chd* *:chdir*
 :chd[ir][!] [path]	Same as |:cd|.
 
+							*:tc* *:tcd*
+:tc[d][!] {path}        Like |:cd|, but set the current directory for the
+                        current tab.  The current directory for other tabs
+                        or the local directory of windows in this tab is not
+                        changed, except for the current window.
+
+							*:tcd-*
+:tcd[!] -		Change to the previous current directory (before the
+			previous ":tcd {path}" command).
+
+							*:tch* *:tchdir*
+:tch[dir][!]		Same as |:tcd|.
+
 							*:lc* *:lcd*
 :lc[d][!] {path}	Like |:cd|, but only set the current directory for the
 			current window.  The current directory for other
-			windows is not changed.
+			windows or any tabs is not changed.
```

Comment:
Instead of a new command (`:tcd`), what if we enabled `:lcd` to take a range, and special-cased  `:0lcd` to set the tab-local working directory? Anyone like this idea, or is it too cryptic?